source/articles/2015/how-to-generate-sales-leads-as-a-freelance-developer.md

```diff
@@ -0,0 +1,223 @@
+* Be Helpful 2 - Forward articles/videos/tweets/etc to people if you think
+  they'll find it interesting / relevant.
+* Don't spend time with people you don't like.
+* Send them articles when you think they may find them interesting. [Here's an
+  old post where I talk about doing it with Google Reader][reader post], which
+tells you how old the post is.
+
+[2]: http://help.followup.cc/knowledge_base/topics/cheat-sheet
+[reader post]: /google-reader-as-a-networking-tool
+
+The goal of these activities
+is to make "business friends"
+whom you like, and who like you.
+If you don't like someone,
+or are just hanging out with them
+because you feel you have to,
+it won't work.
+
+You should be helpful to them,
+recommending candidates to hire,
+articles to read,
+people to meet,
+and services to try.
+
+You should ask them for advice frequently.
+You should ask them for help rarely.
+You should support their businesses,
+retweet their product announcements,
+and support their crowdfunding campaigns,
+_but only do so because you feel like it_.
+
+I should note
+that every job I've ever gotten
+has been a result of networking.
+I'm guessing this is broadly true.
+
+### Why Do This
+
+First of all,
+you make business friends to learn.
+Asking questions helps you understand
+what other companies and situations are like
+which helps you handle your own thing.
+
+Second of all, it feels good to help people,
+or rather it should.
+If it doesn't,
+there's a whole other set of problems to address.
+
+But really,
+the reason you get to know a lot of people
+is to be top of mind when they need help.
+
+An example is:
+Sally works at a company
+and hears
+"oh we need to get help with this Rails app,"
+and thinks
+"I wonder if my friend YOUR_NAME knows someone, or can help..."
+
+Then they call you,
+and you have a discussion about how helpful you can be
+(if at all).
+Be sure to let them know
+if you're the wrong person for the job
+but maybe you know who is.
+
+All of this should feel authentic.
+If it feels slimy or manipulative to you,
+it will seem that way to everyone,
+and you won't build trust (see above).
+
+
+## Content Marketing: Writing
+
+### Specific Tactics I Use
+
+* Set up a blog which clearly states you are hirable and what skills you have.
+* Blog and tweet as regularly as you can about your work.
+  The content should be solid, but quantity and regularity matter much more.
+* If you do something neat for a client, use their name (with permission).
+  They may cross-promote your post for you.
+* Try using an editorial board in Trello to build a pipeline of ideas
+  which you can push along into posts.
+* Set regular writing goals like one post per week, one hour a day, etc.
+* Promote your posts using Twitter, Google+, LinkedIn, Facebook, whatever.
+  You can use [Buffer] to automate this stuff.
+* Find a pal who is good at writing to edit your posts before going out.
+  I have everthing in GitHub and ask for reviews using pull requests.
```

Comment:
Thanks. This is how I spell currently, suggestions welcome for alternatives. efc2f1f

![this is how I spell](https://9dbdc7e4e76730eafd8c-725637a4efb9dbcc26cd49a804f2fb9b.ssl.cf2.rackcdn.com//filer/ouija-board-historical-gallery.png__600x0_q85_upscale.jpg)